### ISSUE TITLE
Issue with SSL

### ISSUE THREAD

**Issue by choungchamnab** [loadimpact/k6#792]
I've generated .pem using these following command : 
`openssl rsa -in private.key -text > private.pem`
`openssl x509 -inform PEM -in certificate.crt > public.pem `

and config in script.js as following 

`export let options = {
  tlsAuth: [
    { domains: ["<url>"], cert: open("public.pem"), key: open("private.pem") }
  ]
};`

but i get following error when trying to run the script : 

`ERRO[0001] GoError: Get https://public.pem?_k6=1: dial tcp: lookup public.pem: no such host `

Below is full code of my script.js 
```js
import http from "k6/http";
import { check,sleep } from "k6";

export let options = {
  tlsAuth: [
    { domains: ["url"], cert: open("public.pem"), key: open("private.pem") }
  ]
};

export default function() {
  //k6 run --vus 100 --duration 60s script.js
  let res = http.get("url");
  check(res, {
    "status was 200": (r) => r.status == 200,
  });
  check(res, {
    "protocol is HTTP/2": (r) => r.proto === 'h2'
  });
  sleep(1);
};
```

**Comment by na--** [comment]
I think you might be running an older k6 version, either `v0.20` or something even older. Newer k6 versions should return a more verbose error that looks like this:
`
ERRO[0000] GoError: The file "public.pem" couldn't be found on local disk, and trying to retrieve it from https://public.pem failed as well. Make sure that you've specified the right path to the file. If you're running k6 using the Docker image make sure you have mounted the local directory (-v /local/path/:/inside/docker/path) containing your script and modules so that they're accessible by k6 from inside of the container, see https://docs.k6.io/v1.0/docs/modules#section-using-local-modules-with-docker. 
`

That said, you shouldn't have gotten either error, and I see at least two k6 bugs here...

The first bug is that k6 doesn't actually open the file - I confirmed it by having the `.pem` files in the same folder as the script and running k6 in from it as well, which produced the above error. The script worked once I added `./` to the beginning of the file names, i.e. `cert: open("./public.pem"), key: open("./private.pem")`, but that shouldn't have been necessary... I've fixed the example in the documentation for now, but your code should have also worked.

The second bug may partially be the cause of the first one - [`open()`](https://docs.k6.io/docs/open-filepath-mode) shouldn't actually try to open URLs, it should only work with local files. I think this happens simply because we [reuse the same `loader`](https://github.com/loadimpact/k6/blob/c665a73e561bcef243b40207640d8099d392f488/js/initcontext.go#L166) that we use for supporting things like `k6 run github.com/loadimpact/k6/samples/http_get.js` and `import`-ing libraries from URLs, so we should fix that.
